packages/wrangler/src/deployment-bundle/bundle-reporter.ts

```diff
@@ -29,7 +30,16 @@ export async function printBundleSize(
 		gzipSize / ONE_KIB_BYTES
 	).toFixed(2)} KiB`;
 
-	logger.log(`Total Upload: ${bundleReport}`);
+	const percentage = (gzipSize / ONE_MIB_BYTES) * 100;
```

Comment:
I think the limit is 10MB now? https://developers.cloudflare.com/workers/platform/limits/#worker-size

Reply:
Annoyingly it's different for free/paid accounts, and there's not any obvious way to tell which limit we should show

Reply:
oh gawd...

Reply:
It might be safer to keep it at 1Mib? Any size over that will become purged from edge colos a lot more quickly when it's not been recently requested, in comparison to smaller scripts, so it's definitely the maximum script size to aim for